openqa-label-known-issues

```diff
@@ -136,31 +138,33 @@ investigate_issue() {
     echo "$reason" >> "$out"
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
         # if we can not even access the page it is something more critical
-        handle_unreachable "$testurl" "$out"
-        # Assume autoinst-log.txt still not found but process job anyway due to reason var
-        if [[ -n $reason ]] && [[ $curl_output = "404" ]] && [[ $reason != null ]]; then
-            label_on_issues_from_issue_tracker "$id"
+        handle_unreachable "$testurl" "$out" || return
```

Comment:
You didn't actually change `handle_unreachable` like in #343 
If it doesn't return `1` then this `return` here is not changing anything.

I suggested that you could just cherry-pick my commit.

Reply:
And this was something I already suggested in the original PR: https://github.com/os-autoinst/scripts/pull/335#discussion_r1711300333

Reply:
`return`  as is here, should return the last exit code, no?

Reply:
What I mean is this and the other `return 1` lines inside of `handle_unreachable.
https://github.com/os-autoinst/scripts/pull/343/files#diff-4b82b9655e797531626dd3da7dce9a5f1e1bcade526ba3ad0262a1a4569a3570R44

The function needs to return 1 to indicate that we should not continue with labeling. Then the `|| return` will be executed.

Reply:
27b7e99